e2e/tests/workspace/workspace-create.spec.ts

```diff
@@ -12,7 +12,7 @@ describe('Creating new workspaces', () => {
     await signIn(page);
   });
 
-  test('Create workspace - NO request for review', async () => {
+  test.skip('Create workspace - NO request for review', async () => {
```

Comment:
is this test broken now?

Reply:
good catch. I forgot to un-skip after local testing.